README.md

```diff
@@ -1,13 +1,21 @@
-## Studio Library
+<p align="center">
+  <a href="http://www.studiolibrary.com/">
+    <img
+      alt="logo"
+      src="./resource/icons/icon_white_on_blue.png"
+    />
+  </a>
+  <h1 align="center">Studio Library</h1>
+  <h4 align="center">Animation and pose manager for Maya</h4>
```

Comment:
Let's remove "Animation and pose manager for Maya" and just have the description.